backend/main.go

```diff
@@ -49,8 +50,13 @@ func main() {
 	router.GET("/cities", handlers.GetCities)
 
 	//JobOffers
-	router.GET("/jobs", handlers.GetJobOffers)
-	router.POST("/jobs", handlers.PostJobOffer)
+	jobsRouter := router.Group("/jobs")
+	{
+		jobsRouter.Use(limits.RequestSizeLimiter(5242880)) // 5MB. if request body is larger than 5MB, it will return 413 error
```

Comment:
Ideally we would like to only limit the size of the image, and not the size of the body as the image body may contain the request payload. The difference is not massive but if we can check solely the size of the image that would be great